src/containers/LandingPage/FallbackPage.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
-import PageBuilder from '../PageBuilder/PageBuilder';
+import loadable from '@loadable/component';
+
 import css from './FallbackPage.module.css';
 
+const PageBuilder = loadable(() => import('../../containers/PageBuilder/PageBuilder'));
```

Comment:
```suggestion
const PageBuilder = loadable(() => import(/* webpackChunkName: "PageBuilder" */ '../../containers/PageBuilder/PageBuilder'));
```

https://webpack.js.org/api/module-methods/#magic-comments

Reply:
Add this also to other dynamic imports (to help customizers pick this naming feature).